jsk_data/src/jsk_data/download_data.py

```diff
@@ -97,8 +97,9 @@ def check_md5sum(path, md5):
     if md5 and len(md5) != 32:
         raise ValueError('md5 must be 32 charactors\n'
                          'actual: {} ({} charactors)'.format(md5, len(md5)))
-    print('[%s] Checking md5sum (%s)' % (path, md5))
-    is_same = hashlib.md5(open(path, 'rb').read()).hexdigest() == md5
+    path_md5 = hashlib.md5(open(path, 'rb').read()).hexdigest()
+    print('[%s] Checking md5sum (%s ... %s)' % (path, path_md5, md5))
```

Comment:
Please print which md5 is requried and which is actual.
e.g.
`Checking md5sum (actuall: %s, required: %s)`

Reply:
resolved in my latest commit.